packages/reactivity/src/ref.ts

```diff
@@ -5,6 +5,7 @@ import { isProxy, toRaw, isReactive, toReactive } from './reactive'
 import type { ShallowReactiveMarker } from './reactive'
 import { CollectionTypes } from './collectionHandlers'
 import { createDep, Dep } from './dep'
+import { IfAny } from 'packages/runtime-core/src/helpers/typeUtils'
```

Comment:
I think we should move `IfAny` to the `shared` module is better. should avoid import it from `runtime-core`

Reply:
Let me do it